index/faults/module/faults.go

```diff
@@ -123,9 +123,6 @@ func (s *Index) Close() error {
 // start is a long running job that keeps the index up to date with chain updates.
 func (s *Index) start() {
 	defer close(s.finished)
-	if err := s.updateIndex(); err != nil {
-		log.Errorf("initial updating faults index: %s", err)
-	}
```

Comment:
Since the Fault Index is disabled, we shouldn't consider firing updating this data on startup.